mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/Layout.kt

```diff
@@ -37,15 +37,16 @@ internal sealed class NoContentMeasureScope {
 
 @Composable
 @MosaicComposable
+@Suppress("ktlint:compose:param-order-check") // Order is correct, check just can't tell.
```

Comment:
Yea, it being a `fun interface` messes with the rule here (as it's only looking at lambdas at face value). I might add an extra check in case there is a typealias or a fun interface that's being referenced here, just to cover more edge cases.

Reply:
That's what I figured. I wasn't sure if it was possible to check for a `fun interface` or not since there's no type resolution.